app/src/ui/diff/text-diff.tsx

```diff
@@ -1004,6 +1009,17 @@ export class TextDiff extends React.Component<ITextDiffProps, ITextDiffState> {
     if (batchedOps.length > 0) {
       cm.operation(() => batchedOps.forEach(x => x()))
     }
+
+    const diffSize = getLineWidthFromDigitCount(
+      this.state.diff.maxLineNumber.toString().length
```

Comment:
Why not use your `getNumberOfDigits` here just for consistency sake?

Reply:
🤦  I missed this one. I did good asking for a second pair of 👀  Thank you!!